docs/setup/memfootprint-test-setup.md

```diff
@@ -0,0 +1,75 @@
+# Measuring memory footprint of EdgeCore
+
+## Why measuring memory footprint
+
+- This platform is also tagged for a light weighted edge computing deployment
+- To be able to be deployed over devices with less resources (for example, 256MB RAM)
+- It is required to know by deploying as many as possible pods, it showcases as much as less possible memory footprint
+
+## KPI's measured
+
+- %CPU
+- %Memory
+- Resident Set Size (RSS)
+
+## How to test
+
+After deployment and provisioning of kubeedge cloud and edge components in 2 VM's (supported and tested over ubuntu 16.04) respectively, start deploying pods from 0 to 100 in steps of 5. Keep capturing above KPI's using standard linux "ps" commands, after each step.
+
+### Test setup
+
+![Kubeedge TestSetup](../../docs/images/memfootprint-img/perftestsetup_diagram.PNG)  
+
+*Fig 1: Kubeedge TestSetup*
+
+### Creating a setup
+
+#### Requirements
+
+- Host machine's or VM's resource requirements can mirror the edge device of your choice
+- Resources used for above setup are 4 CPU, 8GB RAM and 200 GB Disk space. OS is Ubuntu 16.04.
+- Docker image used to deploy the pods in edge, needs to be created. The steps are:
+    1. Go to github.com/kubeedge/kubeedge/edge/hack/memfootprint-test/
+    2. Using the Dockerfile available here and create docker image (`perftestimg:v1`).
+    3. Execute the docker command `sudo docker build --tag "perftestimg:v1" .`, to get the image.
+
+#### Installation
+
+* For kubeedge Cloud and Edge:
+
+    Please follow steps mentioned in Kubeedge README.md
+
+* For docker image:
+
+- Deploy docker registry to either edge on any VM or host which is reachable to edge. Follow the steps mentioned here: https://docs.docker.com/registry/deploying/
+- Create `perftestimg:v1` docker image on the above mentioned host
+- Then push this image to docker registry using "docker tag" and "docker push" commands (Refer: Same docker registry url mentioned above)
```

Comment:
nit: use code highlighting for commands instead of double quotes.
"docker tag" -> `docker tag`
"docker push" -> `docker push`